lib/FFI/CheckLib.pm

```diff
@@ -57,6 +57,36 @@ our $system_path = [];
 our $os ||= $^O;
 my $try_ld_on_text = 0;
 
+sub _homebrew_lib_path {
+  return () unless (qx`command -v brew`)[0];
+  chomp(my $brew_path = (qx`brew --prefix`)[0]);
+  return "$brew_path/lib";
+}
+
+sub _macports_lib_path {
+  my $port_path = (qx`command -v port`)[0];
+  return () unless $port_path;
+  chomp($port_path);
+  return $port_path =~ s|bin/port|lib|r;
```

Comment:
Sorry but we can't use `/r` for this module, because that bumps the required Perl to 5.14

```
% perlver lib --blame                

 ------------------------------------------------------------
 File    : lib/FFI/CheckLib.pm
 Line    : 70
 Char    : 24
 Rule    : _regex
 Version : 5.013002
 ------------------------------------------------------------
 s|bin/port|lib|r
 ------------------------------------------------------------
```